WordPress/Classes/Utility/Analytics/WPAnalyticsEvent.swift

```diff
@@ -101,6 +109,18 @@ import Foundation
             return "gutenberg_unsupported_block_webview_shown"
         case .gutenbergUnsupportedBlockWebViewClosed:
             return "gutenberg_unsupported_block_webview_closed"
+        case .pushNotificationsPrimerSeen:
+            return "notifications_primer_seen"
+        case .pushNotificationsPrimerAllowTapped:
+            return "notifications_primer_allow_tapped"
+        case .pushNotificationsPrimerNoTapped:
+            return "notifications_primer_no_tapped"
+        case .secondNotificationsAlertSeen:
+            return "second_notifications_alert_seen"
```

Comment:
Just wondering if we should swap the event name around here so it's similar to the `notifications_primer` events – for example, `notifications_second_alert_seen`. It might help out when looking at tracks events and trying to filter them?

Reply:
sure!!